pkg/network/pod/annotations/generator.go

```diff
@@ -98,6 +102,37 @@ func (g Generator) GenerateFromSource(vmi *v1.VirtualMachineInstance, sourcePod
 	return annotations, nil
 }
 
+func (g Generator) GenerateFromPod(vmi *v1.VirtualMachineInstance, pod *k8Scorev1.Pod) (map[string]string, error) {
+	annotations := map[string]string{}
+
+	netInfoAnnotation, err := g.generateNetworkInfoAnnotation(vmi, pod)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This is a logic change which has been introduced without explanation.
The logic has ignored such errors and just logged them, resulting on the whole reconcile not to fail but to just skip the network info update.

Could you please reason on why this change needs to do this?